pom.xml

```diff
@@ -70,6 +70,16 @@
 			<groupId>org.hibernate.validator</groupId>
 			<artifactId>hibernate-validator</artifactId>
 		</dependency>
+		<dependency>
+			<groupId>io.springfox</groupId>
+			<artifactId>springfox-swagger-ui</artifactId>
+			<version>3.0.0</version>
+		</dependency>
```

Comment:
Why required that dependency?

Reply:
agree

Reply:
I forgot to remove it after fixed some bugs